server/leafnode.go

```diff
@@ -1696,7 +1702,7 @@ func (c *client) forceAddToSmap(subj string) {
 	defer c.mu.Unlock()
 
 	n := c.leaf.smap[subj]
-	if n != 0 {
+	if n != 0 || c.leaf.smap == nil {
```

Comment:
If would have personally do the check and return if that is the case, before the actual lookup.

Reply:
not sure I can do that. the loop iterates over account clients

Reply:
Just meant something like this:

Reply:
sorry misread that as loop not lookup. will fix